core_api/src/semantic_routes.py

```diff
@@ -0,0 +1,107 @@
+from semantic_router import Route
+from semantic_router.encoders import HuggingFaceEncoder
+from semantic_router.layer import RouteLayer
+
+from redbox.model_db import MODEL_PATH
+
+# === Pre-canned responses for non-LLM routes ===
+INFO_RESPONSE = """
+I am RedBox, an AI focused on helping UK Civil Servants, Political Advisors and
+Ministers triage and summarise information from a wide variety of sources.
+"""
+
+ABILITY_RESPONSE = """
+* I can help you search over selected documents and do Q&A on them.
```

Comment:
our markdown formatter in the front end is interpreting:

```
long lines of text that go on and on and on and have new line continuation mark like this \
in them as code and assigns them the <code> tag which isnt what we want
```
![image](https://github.com/i-dot-ai/redbox-copilot/assets/8233643/da07abad-789a-4cb9-aff8-c6a6335b07a6)

so i have stripped them out